### PR TITLE
CSharp Documentation with working anchor link within document

### PR DESCRIPTION
**Pull request by lunat**
Added anhors to csharp client documentation

### CODE REVIEW

**Comment by wing328** [comment]
@lunat thanks for the PR. The change looks good to me and I'll merge the PR later today.

File: modules/swagger-codegen/src/main/resources/csharp/README.mustache [hunk 1]
```diff
@@ -126,6 +128,7 @@ No model defined in this package
 
 ## Documentation for Authorization
 
+<a name="{{#authMethods}}{{name}}"></a>
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/csharp/README.mustache]: This line is causing issue due to missing close tag `{{/authMethods}}` and I've removed it in my PR.

**Comment by lunat** [comment]
@wing328 could you please try this code:

`<a name="{{name}}"></a>`

**Comment by wing328** [comment]
@lunat can you please pull the latest master and submit a PR with the mentioned change?

(we've setup Shippable CI to catch the invalid mustache template moving forward)

**Pull request merged**